test/SchemaField_test.js

```diff
@@ -137,4 +137,46 @@ describe("SchemaField", () => {
         .to.have.length.of(1);
     });
   });
+
+  describe("description support", () => {
+    const schema = {
+      type: "object",
+      properties: {
+        foo: {type: "string", description:"A Foo field"},
+        bar: {type: "string"}
+      }
+    };
+    it("should render description if available from the schema", () => {
+      const {node} = createFormComponent({schema});
+      expect(node.querySelectorAll("div#root_foo__description"))
+        .to.have.length.of(1);
+    });
+
+    it("should render description if available from a referenced schema", () => {
+      // Overriding.
+      const schema = {
```

Comment:
I'm surprised you can reassign a `const` here. You should probably name this specific schema differently instead.